qrcode/main.py

```diff
@@ -228,7 +222,7 @@ def best_fit(self, start=None):
             util.BIT_LIMIT_TABLE[self.error_correction], needed_bits, start
         )
         if self.version == 41:
-            raise exceptions.DataOverflowError()
+            raise exceptions.DataOverflowError
```

Comment:
Might be better to actually add a suitable message?

Reply:
Yes! I started to collect the various tickets related to this. Ideally there's a specific hint how and why data overflows. Its not just a simple value, as it varies depending on the type of QR, error correction level etc. I'll handle that separately.

https://github.com/lincolnloop/python-qrcode/issues?q=is%3Aissue%20state%3Aopen%20label%3Aerror%3Adata-overflow